electron/settings.html

```diff
@@ -290,7 +290,7 @@ <h3>Session Win/Loss Counters</h3>
       <div id="theme-pane" value="theme" rv-showsettingspane="settingsPaneIndex">
         <div class="settings-container">
           <div class="settings-row">
-            <input rv-checked="useFlat" id="enable-flat-theme-toggle" type="checkbox" data-toggle="toggle"> Enable flat theme <span class="helper-text">Changes card display to a simpler "flat" display
+            <input rv-checked="useFlat" id="enable-flat-theme-toggle" type="checkbox" data-toggle="toggle"> Enable flat theme <span class="helper-text">Changes card display to a simpler "flat" display</span>
```

Comment:
Thx for the catch.

Reply:
> Maybe also set name, not just abbreviation?

I can do one better: 
![image](https://user-images.githubusercontent.com/1424566/48962820-a9295500-ef3a-11e8-9f2c-f8b4b30dd351.png)

Turns out these images are in the WotC media kit and should be fair game to use.

Reply:
Nice!

Reply:
haha, not sure why this comment ended up here, but :shrug: